newsfragments/649.feature.md

```diff
@@ -0,0 +1,58 @@
+# Features
+
+## Support local constant
+
+Example:
+```python
+contract Foo:
+    pub fn bar():
+        const LOCAL_CONST: i32 = 1
+```
+
+## Support constant expression
+
+Example:
+```python
+const GLOBAL: i32 = 8
+
+contract Foo:
+    pub fn bar():
+        const LOCAL: i32 = GLOBAL * 8
+```
+
+## Support constant generics expression
+
+Example:
+```python
+const GLOBAL: u256= 8
+const USE_GLOBAL: bool = false
+type MY_ARRAY = Array<i32, { GLOBAL / 4 }>
+
+contract Foo:
+    pub fn bar():
+        let my_array: Array<i32, { GLOBAL if USE_GLOBAL else 4 }>
+```
+
+# Bug fixes
```

Comment:
Just a nitpick but next time it would be better to create another file `649.bugfix.md` because our changelog generator picks up these file name keywords to group the items accordingly. No big deal though, we can always apply changes to the changelog by hand.

Reply:
Oh, Ok. I understand.